apps/fipsinstall.c

```diff
@@ -528,6 +537,9 @@ int fipsinstall_main(int argc, char **argv)
         case OPT_DISALLOW_SIGNATURE_X931_PADDING:
             fips_opts.sign_x931_padding_disabled = 1;
             break;
+        case OPT_PBKDF2_LOWER_BOUND_CHECK:
+            fips_opts.pbkdf2_lower_bound_check = 1;
```

Comment:
This should be 0, as there is no way to turn this off.. I think this needs to be a NO_ like option. (See NO_SECURITY_CHECKS)

Reply:
i.e. It defaults to the value of 1 and the pendantic option sets it to 1 also..

Reply:
Changed `pbkdf2_lower_bound_check` option to `no_pbkdf2_lower_bound_check` option.